api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -668,6 +669,13 @@ public void testGetStatus() {
     assertThat(bufferStatus.getBufferSize()).isEqualTo(numberAvailable);
   }
 
+  @Test
+  public void testGetGaugeData() {
+    final Map<OpenCensusStatsViewInfo, Number> result = billingProjectBufferService.getGaugeData();
+    assertThat(result).hasSize(6);
```

Comment:
Opt: What is the number six here? Does it make sense to capture it in a variable so there's some documentation?

Reply:
Yeah, this is a really fragile test. It's asserting that there are as many as there were the first time I ran the test.

I thought about having a system where each service or whatever could have its own enum for metrics, but that's hard to do cleanly when I want nontrivial metadata on each enum value. But if I had that, I'd make this be the size of the right one.